### PR TITLE
Angular update9 to12

### PR DESCRIPTION
**Pull request by Rikveet**
# Changes 🛠

## Any new npm dependencies?

NO

# Testing 🧪

Changes from the latest PR are deployed to https://brockcsc-pr.web.app once the checks are complete. Can use this for testing.

### Any other info neededed for testing?

NO

### CODE REVIEW

File: src/app/views/exec-applications/execapplications.component.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      email: ['', [Validators.email, Validators.required]],
+      intrCscExec: ['', Validators.required],
+      skills: ['', Validators.required],
+      workshop: ['', Validators.required],
+      years: ['', Validators.required],
+    });
+
+    this.submitted = false;
+    this.googleForm = {
+      url: 'https://docs.google.com/forms/d/e/1FAIpQLSfBrVc9NN552xvMBIQ36_-q8jUuuMvgNdBVn2fTM4bsgkcRqw/formResponse',
```
Comment (JoelGritter) [inline src/app/views/exec-applications/execapplications.component.ts:28]: Intentionally left the URL hardcoded?

**Comment by kindeep** [comment]
I would remove link from the top nav, instead maybe a link in /team saying "Apply to be an Exec"

**Review by kindeep — CHANGES_REQUESTED** [review]
Remove link from nav, add link to the /team page saying “Apply to be an exec”. Add another link in the signup page, just at the bottom I think. Maybe also add a link on homepage, somewhere around the “Learn about the team” button

**Review by kindeep — APPROVED** [review]
LGTM :shipit:

**Pull request merged**